crypto/keyring/keyring.go

```diff
@@ -64,13 +64,16 @@ type Keyring interface {
 	Delete(uid string) error
 	DeleteByAddress(address sdk.Address) error
 
-	// NewMnemonic generates a new mnemonic, derives a hierarchical deterministic
-	// key from that, and persists it to the storage. Returns the generated mnemonic and the key
-	// Info. It returns an error if it fails to generate a key for the given algo type, or if
+	// NewMnemonic generates a new mnemonic, derives a hierarchical deterministic key from that, and
+	// persists it to the storage. Returns the generated mnemonic and the key Info secured with the given passphrase.
+	// It returns an error if it fails to generate a key for the given algo type, or if
 	// another key is already stored under the same name.
-	NewMnemonic(uid string, language Language, hdPath string, algo SignatureAlgo) (Info, string, error)
+	//
+	// NOTE: a passphrase set to 'default' will set the passphrase to the DefaultBIP39Passphrase value.
```

Comment:
added this logic to allow for empty string passphrases. cc @alessio

Reply:
Please remove the NOTE: prefix. It does not end up being displayed in any particular way :+1: